code/game/objects/effects/decals/Cleanable/humans.dm

```diff
@@ -52,14 +52,12 @@ var/global/list/blood_list = list()
 
 	if(ticker && ticker.mode && ticker.mode.name == "cult")
 		var/datum/game_mode/cult/mode_ticker = ticker.mode
-		if((mode_ticker.objectives[mode_ticker.current_objective] == "bloodspill") && !mode_ticker.narsie_condition_cleared)
-			var/turf/T = get_turf(src)
-			if(T && (T.z == map.zMainStation))//F I V E   T I L E S
-				if(locate("\ref[T]") in mode_ticker.bloody_floors)
-				else
-					mode_ticker.bloody_floors += T
-					mode_ticker.bloody_floors[T] = T
-					mode_ticker.blood_check()
+		var/turf/T = get_turf(src)
+		if(T && (T.z == map.zMainStation))//F I V E   T I L E S
+			if(!(locate("\ref[T]") in mode_ticker.bloody_floors))
```

Comment:
Well for the record I tried a lot of different versions of this code when I first coded the bloody floors and this one was the only one that worked properly, that's why I'm really worried that changing it would break the objective.